### PR TITLE
add Blog

### PR DESCRIPTION
**Pull request by Melinx**

### CODE REVIEW

File: staff/Melinx/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+        </figure>
+        <img src="https://daks2k3a4ib2z.cloudfront.net/55fba7ade2ae0be15df2b309/58533c951ed34e800f975234_adding-html5-semantic-elements.png"></img>
+    
+
+
+    </main>
+</body>
+
```
Comment (PacoGD) [inline staff/Melinx/blog/index.html:64]: You should have a footer.

File: staff/Melinx/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <section>🔲 Section ONE - </section>
+            <section>🔳 Section TWO - </section>
+           
+        </article>
+
+        <ul>
+            <li id="sup"> Check this site for a good start:</li>
+            <a href="https://internetingishard.com/html-and-css/semantic-html/" target="blank">HTML & CSS | Interesting is Hard</a>
+
+            <li id="sup">Try analyzing Element tab from some website eg:</li>
```
Comment (EzequielBurgosL) [inline staff/Melinx/blog/index.html:41]: Why have you named the id "sup"? Is it for semantic purposes? 🗡

File: staff/Melinx/blog/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog</title>    
+    <link rel="icon" href="images/basic-red-heart.png" type="image/png" sizes="16x16">
+
+</head>
+
+<body>
+    <main>
```
Comment (PacoGD) [inline staff/Melinx/blog/index.html:14]: You should have a header.

File: staff/Melinx/blog/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <h1>~ How to Make a Nicely Sturctured Blog ~</h1>
+
+        <h3>HINTS shared in Slack:</h3>
+        <ul>
+            <li>ONE! <a href="https://www.hongkiat.com/blog/html-5-semantics/" target="blank">A Look Into Proper HTML5 Semantics</a></li>
+            <li>TWO! <a href="https://webflow.com/blog/html5-semantic-elements-and-webflow-the-essential-guide"target="blank"> ...and Webflow: the essential guide</a></li>
+            
+        </ul>
+        <article>
+            The article element represents a complete, or self-contained, composition in a document, page, application, or site and that
```
Comment (EzequielBurgosL) [inline staff/Melinx/blog/index.html:24]: This text should contained inside a Tag (f. ex. in a `<p>`)
`<article>`
  `<p>bla bla bla...</p>`
`</article>`

File: staff/Melinx/blog/index.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        <h2>Let's review some basic structure elements:</h2>
+        <p>Paragraphs SHOULD NOT be inside a li element.</p>
+        <ul>
+            <li id="sup">Headings can be named with numbers like h1, h2, etc, depending on sizes, I think.</li>
+            <li id="sup">Paragraphs.. the 'p' element is used to represent a paragraph.  </li>
+        </ul>
+
+        <h3>Let's see how images work...(figure)</h3>
+        <figure>
+            <img src="basic-red-heart" alt="blablabla">
```
Comment (agandia9) [inline staff/Melinx/blog/index.html:54]: no title attr here...

File: staff/Melinx/blog/index.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+
+        </figure>
+        <img src="https://daks2k3a4ib2z.cloudfront.net/55fba7ade2ae0be15df2b309/58533c951ed34e800f975234_adding-html5-semantic-elements.png"></img>
```
Comment (agandia9) [inline staff/Melinx/blog/index.html:58]: no alt or/and title here...

**Pull request merged**